lms/envs/common.py

```diff
@@ -735,7 +735,7 @@
 
 TRACKING_SEGMENTIO_WEBHOOK_SECRET = None
 TRACKING_SEGMENTIO_ALLOWED_TYPES = ['track']
-TRACKING_SEGMENTIO_DISALLOWED_SUBSTRING_NAMES = ['.bi.']
+TRACKING_SEGMENTIO_DISALLOWED_SUBSTRING_NAMES = []
```

Comment:
The transformer filters them out [here](https://github.com/edx/edx-platform/pull/15237/files#diff-ae377d272298160090ef79e001e120feR63).